### PR TITLE
unified processing parse env max_inflight, ack_wait variable

### PR DESCRIPTION
**Pull request by feifeiiiiiiiiiii**
Signed-off-by: qinpengfei <qinpengfei@jd.com>

## Description

Describe  #38 
## Motivation and Context

- [x] I have raised an issue to propose this change ([required](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md))

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/openfaas/faas/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: readconfig.go [hunk 1]
```diff
@@ -52,4 +76,6 @@ type QueueWorkerConfig struct {
 	FunctionSuffix string
 	DebugPrintBody bool
 	WriteDebug     bool
+	maxInflight    int
```
Comment (alexellis) [inline readconfig.go]: I would like these to match the style of the other variables.

**Comment by alexellis** [comment]
Thanks for this PR 👍 I have left a comment for you.

File: readconfig.go (hunk 1 above)
Comment (feifeiiiiiiiiiii) [reply readconfig.go]: @alexellis ok, I will modify

**Comment by alexellis** [comment]
@LucasRoesler please could I ask you to give this a review? What I'd like is to be sure there is no unintended typo or regression made by the refactoring.

File: readconfig_test.go [hunk 2]
```diff
@@ -44,4 +47,50 @@ func Test_ReadConfig(t *testing.T) {
...
+
+	expectedMaxInflight = 1
+	if config.MaxInflight != expectedMaxInflight {
+		t.Logf("Expected maxInflight `%v` actual `%v`\n", expectedMaxInflight, config.MaxInflight)
+		t.Fail()
+	}
+
+	expectedAckWait = time.Second * 30
+	if config.AckWait != expectedAckWait {
+		t.Logf("Expected maxInflight `%v` actual `%v`\n", expectedAckWait, config.AckWait)
```
Comment (LucasRoesler) [inline readconfig_test.go]: The language on this error and a couple others need to be updated. The message says "maxInFlight" but it should be "ackWait"

**Comment by alexellis** [comment]
Much appreciated.

File: readconfig_test.go (hunk 2 above)
Comment (feifeiiiiiiiiiii) [reply readconfig_test.go]: @LucasRoesler my mistake, I will fixed it

**Comment by alexellis** [comment]
Great, thanks for putting your name to this Lucas. I'll merge since you've given it the rubber stamp.

**Pull request merged**